cli/build.rs

```diff
@@ -104,6 +104,18 @@ fn ts_version() -> String {
     .collect::<String>()
 }
 
+fn git_commit_hash() -> String {
+  let output = std::process::Command::new("git")
+    .arg("rev-list")
+    .arg("-1")
+    .arg("HEAD")
+    .output()
+    .expect("failed to execute process");
+  std::str::from_utf8(&output.stdout[..7])
+    .unwrap()
+    .to_string()
+}
+
```

Comment:
Is this going to work when doing `cargo install deno`?

Reply:
It's a good point. I think it will break "cargo install deno".

Reply:
I see. It won't work with `cargo install`...

We need a different strategy here.

Reply:
It seems sha1 is stored in the file called `.cargo_vcs_info.json` in crates.io. (ref: https://github.com/rust-lang/cargo/issues/5629 ) Maybe we can use it.

Reply:
Maybe just return `"crates.io"` if anything in `git_commit_hash` fails?